app/jobs/consume_assessments_job.rb

```diff
@@ -1,212 +1,156 @@
-                                        # Remove values so that the values will appear as blank in a symptomatic report
-                                        # this will indicate that the person needs to be reached out to to get the actual values
-                                        threshold_condition.clone_symptoms_remove_values
-                                      else
-                                        # The person is not experiencing symptoms, we can infer that the bool symptoms are the opposite
-                                        # of the threshold values that represent symptomatic
-                                        threshold_condition.clone_symptoms_negate_bool_values
-                                      end
-            reported_condition = ReportedCondition.new(symptoms: typed_reported_symptoms, threshold_condition_hash: message['threshold_condition_hash'])
-            assessment = Assessment.new(reported_condition: reported_condition, patient: dependent)
+    # Invalid threshold_condition_hash
+    if threshold_condition.nil?
+      log_and_capture("ConsumeAssessmentsJob: No ThresholdCondition found (patient: #{patient.id}," \
+                      " threshold_condition_hash: #{message['threshold_condition_hash']})")
+      return
+    end
 
-            # If current user in the collection of patient + patient dependents is the patient, then that means
-            # that they reported for themselves, else we are creating an assessment for the dependent and
-            # that means that it was the proxy who reported for them
-            assessment.who_reported = patient.submission_token == dependent.submission_token ? 'Monitoree' : 'Proxy'
-            begin
-              reported_condition.transaction do
-                reported_condition.save!
-                assessment.symptomatic = assessment.symptomatic? || message['experiencing_symptoms']
-                assessment.save!
-              end
-            rescue ActiveRecord::RecordInvalid => e
-              Rails.logger.info(
-                "AssessmentsController: Unable to save assessment due to validation error for patient ID: #{dependent.id}. " \
-                "Error: #{e}"
-              )
-            end
-            queue.commit
-          end
-        end
-      rescue JSON::ParserError
-        Rails.logger.info 'ConsumeAssessmentsJob: skipping invalid message...'
-        queue.commit
-        next
+    if message['reported_symptoms_array']
+      typed_reported_symptoms = Condition.build_symptoms(message['reported_symptoms_array'])
+      reported_condition = ReportedCondition.new(symptoms: typed_reported_symptoms, threshold_condition_hash: message['threshold_condition_hash'])
+      assessment = Assessment.new(reported_condition: reported_condition, patient: patient, who_reported: 'Monitoree')
+      assessment.symptomatic = assessment.symptomatic? || message['experiencing_symptoms']
```

Comment:
wouldn't it just be this since `experiencing_symptom` is for the generic channel case?

```suggestion
      assessment.symptomatic = assessment.symptomatic?
```

Reply:
Same as my response on your other comment - this part of the code isn't even accurate. Not sure why this was the case.

Reply:
6c819e7